lib/runtime-core/src/error.rs

```diff
@@ -1,13 +1,22 @@
+//! The error module contains the data structures and helper functions used to implement errors that
+//! are produced and returned from the wasmer runtime core.
 use crate::types::{FuncSig, GlobalDescriptor, MemoryDescriptor, TableDescriptor, Type};
 use core::borrow::Borrow;
 use std::any::Any;
 
+/// Aliases the standard `Result` type as `Result` within this module.
```

Comment:
This comment is really a restatement of the code, when it should be a description of what this type is used for. Maybe something like "Result of an attempt to compile the provided WebAssembly module into a `Module`" to copy from the definition of `CompileError`.

Reply:
I've updated these aliases using your suggestion.